pages/common/zotero.md

```diff
@@ -0,0 +1,20 @@
+# Zotero
```

Comment:
```suggestion
# zotero
```

Reply:
I actually think it should remain capitalised as it is not just a command but a platform by itself